core/lib/vm_utils/Cargo.toml

```diff
@@ -11,12 +11,11 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
-zksync_multivm.workspace = true
-zksync_types.workspace = true
+zksync_contracts.workspace = true
 zksync_dal.workspace = true
-zksync_state.workspace = true
+zksync_types.workspace = true
+zksync_vm_interface.workspace = true
```

Comment:
With this change, `zksync_vm_utils` now feels even more bogus. Do you think it'll be possible to merge it with the newly added crate?
The main complexity seems to be the fact that it's glued to `zksync_dal`, but if you look at it, it seems to be mostly used to fetch VM inputs. Given that `zksync_state` already provides an implementation for the Postgres-backed storage, maybe it can be solved by introducing a new abstraction (e.g. `VmBootstrap`) that will have a single Postgres-backed implementation in `zksync_state` crate.

Not proposing doing it in this PR though.

Reply:
I certainly want to remove `zksync_vm_utils` in some way; as you've said, not in this PR.